tests/Backend/Teradata/ImportTypedTableTest.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace Keboola\Test\Backend\Teradata;
+
+use Keboola\Csv\CsvFile;
+use Keboola\StorageApi\ClientException;
+use Keboola\StorageApi\Workspaces;
+use Keboola\TableBackendUtils\Column\Teradata\TeradataColumn;
+use Keboola\Test\Backend\Workspaces\Backend\WorkspaceBackendFactory;
+use Keboola\Test\Backend\Workspaces\ParallelWorkspacesTestCase;
+
+class ImportTypedTableTest extends ParallelWorkspacesTestCase
+{
+    public function setUp(): void
+    {
+        parent::setUp();
+
+        $token = $this->_client->verifyToken();
+
+        if (!in_array('tables-definition', $token['owner']['features'])) {
```

Comment:
presne na tomto vyfailovaly vsechny testy v tagu . https://dev.azure.com/keboola-dev/26bf9dc4-aef5-4354-84fe-c52ee81d4cfa/_apis/build/builds/27939/logs/47 prosim pridej to do `ParatestBackendTeradataCmd` a otoc projekt na testingu

Reply:
jj, vim, jdu na to